### PR TITLE
LEB-4068: Drupal 9 readiness.

### PR DESCRIPTION
**Pull request by nidaismailshah**

### CODE REVIEW

**Comment by scor** [comment]
Hi @nidaismailshah, your PR looks good, however could you explain why you didn't include the last 3 hunks from the patch on drupal.org (https://www.drupal.org/project/acquia_lift/issues/3095278#comment-13366152)?
```
diff --git a/tests/src/Unit/Service/Context/PageContextTest.php b/tests/src/Unit/Service/Context/PageContextTest.php
diff --git a/tests/src/Unit/Service/Context/PathContextTest.php b/tests/src/Unit/Service/Context/PathContextTest.php
diff --git a/tests/src/Unit/Service/Helper/PathMatcherTest.php b/tests/src/Unit/Service/Helper/PathMatcherTest.php
```
All these changes are about switching from $this->getMock() to $this->createMock() which seems consistent with https://www.drupal.org/project/drupal/issues/2929133

**Comment by scor** [comment]
Also if you want to upload your patch on drupal.org for consistency, feel free to do so, as I'm planning to credit all the people who participated in that issue.

**Comment by nidaismailshah** [comment]
@scor Added the patch to drupal.org issue.
Regarding the changes in above files, I think I have included these. Can you point me to what I have missed?

File: acquia_lift.services.yml [hunk 1]
```diff
@@ -7,6 +7,6 @@ services:
     arguments: ['@config.factory', '@path.current', '@request_stack', '@acquia_lift.service.helper.path_matcher']
   acquia_lift.service.helper.path_matcher:
     class: Drupal\acquia_lift\Service\Helper\PathMatcher
-    arguments: ['@path.alias_manager', '@path.matcher']
+    arguments: ['@path_alias.manager', '@path.matcher']
```
Comment (scor) [inline acquia_lift.services.yml:10]: it seems this is breaking our test in Drupal 8.7, so we might have to require Drupal 8.8 in our .info.

Comment (scor) [reply acquia_lift.services.yml:10]: cc @nidaismailshah

**Comment by scor** [comment]
Merged with https://github.com/acquia/acquia_lift/commit/4e5a14666e29210c6229ac152e7ea6c7e89fe613

**Pull request closed without merging**